tools/bin/goreleaser_utils

```diff
@@ -6,6 +6,9 @@ set -xe
 before_hook() {
   local -r lib_path=tmp
 
+  echo $(ls)
```

Comment:
Sorry, does it make sense to convert this PR to the draft state before it goes out for a review?

Reply:
Unfortunately I don't have a way to put this back into draft. Took it out of it since I think it was interfering with how CI was running. I'm picking this back up today so should be ready for an actual review shortly.